@xen-orchestra/lite/lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  "*.{ts,vue}": () => "vue-tsc --noEmit",
+};
```

Comment:
From what I understand, this file is shadowing the root config and will break Prettier, ESLint and the automatic test runs.

You should probably import and extend the root config instead.

Reply:
Or you can simply add this command to the `test` script of this package :slightly_smiling_face: 

https://github.com/vatesfr/xen-orchestra/commit/09a2f45ada47582239ee8e1e5da7fad1d3b10807

Reply:
It doesn't seem easy to just import the root config since it is configured to call a local script (`scripts/run-changed-pkgs.js`, which therefore does not exist in the current directory).

Adding the command to the `test` script is a possibility, but then it will run every time, even if no corresponding file is staged (for example, staging a `.md` file or a `.vue` file will run the typescript check).

Can be a temporary workaround, since type-check could be considered as a "test" command.

Another solution would be to extract the common configuration of root lint-staged configuration to a base file, then import it where needed.

Reply:
Maybe I have a better solution.

Just moving the config from `package.json` to `lint-staged.config.js` then replacing
```
"*": "scripts/run-changed-pkgs.js test"
```

With:
```
'*': `${path.join(__dirname, 'scripts/run-changed-pkgs.js')} test`
```

Then it works everywhere.